crates/blockchain-tree/src/lib.rs

```diff
@@ -1,3 +1,9 @@
+#![cfg_attr(docsrs, feature(doc_cfg))]
+#![doc(
+    html_logo_url = "https://raw.githubusercontent.com/paradigmxyz/reth/main/assets/reth-docs.png",
+    html_favicon_url = "https://avatars0.githubusercontent.com/u/97369466?s=256",
```

Comment:
@onbjerg you might want to also update this url :)

Reply:
I don't think it scales well to that size, so I opted to keep the Paradigm logo there. cc @gakonst for thoughts?